commanditemmodel.cpp

```diff
@@ -158,6 +161,14 @@ bool CommandItemModel::lessThan(const QModelIndex &left, const QModelIndex &righ
             || (pos_left == pos_right && QSortFilterProxyModel::lessThan(left, right));
     }
 
+    // compare visible names
+    if (leftItem != nullptr && righItem != nullptr)
+    {
+        int comp = leftItem->title().compare(righItem->title());
+        if (comp != 0)
+            return comp < 0;
```

Comment:
Does it hurt we always add brackets after "if"? Please (I'm begging you :smiling_face_with_three_hearts:) take it into style guide (if any).